frontend/src/tests/user-permission-tests.js

```diff
@@ -14,11 +14,43 @@ import { databaseSeeder } from "./setup";
 export function userPermissionsTests(api) {
     const { apiGET, apiPOST } = api;
 
-    const newUserData = {
+    const newInstructorData = {
         utorid: "userxx",
         roles: ["instructor"],
     };
 
+    const newTAData = {
+        utorid: "tauser",
+        roles: ["ta"],
+    };
+
+    const newTAInstructorData = {
+        utorid: "tainstuser",
+        roles: ["ta", "instructor"],
+    };
+
+    async function setupUserPerm(userData) {
```

Comment:
Don't use abbreviations unless they're more common than the full word. E.g., `Perm`->`Permissions`

Reply:
The name of this function doesn't clue me in on what's happening. Try again.

Reply:
Function is missing a doc string. All utility functions should have doc strings. (Look in `utils.js` for examples). You can install the `Document This` extension for VSCode to make creating doc strings easier.